apps/core/client/queries/get-address-form-fields.ts

```diff
@@ -0,0 +1,45 @@
+import { cache } from 'react';
+
+import { client } from '..';
+import { FORM_FIELDS_FRAGMENT } from '../fragments/form-fields';
+import { FormFieldFiltersInput, FormFieldSortInput } from '../generated/graphql';
+import { graphql } from '../graphql';
+import { revalidate } from '../revalidate-target';
+
+interface AddressFormFields {
```

Comment:
I think that after switching to gql.tada we can do it smth like:

```
import { graphql, VariablesOf } from '../graphql';

type Variables = VariablesOf<typeof GET_ADDRESS_FORM_FIELDS>;

interface AddressFormFields {
filters: Variables['Filters'];
sortBy: Variables['sortBy'];
}

```

Reply:
yeah, missed that update, good catch 👍